pkg/session/bootstrap.go

```diff
@@ -1375,6 +1381,7 @@ var (
 		upgradeToVer216,
 		upgradeToVer217,
 		upgradeToVer218,
+		upgradeToVer239,
```

Comment:
Please consult that why the version is changed from 218 to 239, not 219?

Reply:
see the comment
```
	// ...
	// [version219, version238] is the version range reserved for patches of 8.5.x
	// ...

	// next version should start with 239
```